pl_examples/models/allennlp_template.py

```diff
@@ -0,0 +1,214 @@
+import itertools
+import statistics
+from copy import deepcopy
+from typing import Dict, Union
+
+import allennlp.training.util as training_util
+import flatten_dict
+import torch
+from allennlp.data.dataloader import DataLoader
```

Comment:
`allennlp==0.9.0 ` does not have this module

Reply:
This was written for allennlp 1.0 (currently at RC5)